src/schema/posts.ts

```diff
@@ -731,7 +731,10 @@ export const typeDefs = /* GraphQL */ `
       Content of the post (max 4000 chars)
       """
       content: String
-    ): Post! @auth
+    ): Post!
+      @auth
+      @rateLimit(limit: 10, duration: 3600)
+      @highRateLimit(limit: 1, duration: 600)
```

Comment:
Seems off to have two rate limits applied in the same time? User will always hit the high rate limit first and that will mostly remove ability to trigger normal one?

Reply:
The high rate limit only does "rate" calculations if the post is in a high rated squad, in this case Watercooler.